vendor/code.gitea.io/git/commit.go

```diff
@@ -198,8 +198,8 @@ func (c *Commit) CommitsBeforeUntil(commitID string) (*list.List, error) {
 }
 
 // SearchCommits returns the commits match the keyword before current revision
-func (c *Commit) SearchCommits(keyword string) (*list.List, error) {
-	return c.repo.searchCommits(c.ID, keyword)
+func (c *Commit) SearchCommits(keyword string, all bool) (*list.List, error) {
+	return c.repo.searchCommits(c.ID, keyword, all)
```

Comment:
You should change this package directly in the vendor folder.

You should send PR to https://github.com/go-gitea/git first, and after aproved use [govendor][govendor] to update the changes here.

[govendor]: https://github.com/kardianos/govendor